.github/workflows/rumble-vulnerability-data.yaml

```diff
@@ -4,44 +4,69 @@ on:
     - cron: "1 5 * * *"
   workflow_dispatch:
   push:
-    branches: [rumble-vulnerability-data]
+    branches: [rumble-insights]
+
+env:
```

Comment:
If these secrets are scoped to this workflow, any chance we can leverage [environment secrets](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets) instead of environment variables here?

Reply:
I think so! We don't have environments set up for Academy yet, but it makes sense to move to using them.